compat/maven-artifact/src/main/java/org/apache/maven/artifact/versioning/ComparableVersion.java

```diff
@@ -36,25 +36,25 @@
  * <p>
  * Features:
  * <ul>
- * <li>mixing of '<code>-</code>' (hyphen) and '<code>.</code>' (dot) separators,</li>
- * <li>transition between characters and digits also constitutes a separator:
+ * <li>Mixing of '<code>-</code>' (hyphen) and '<code>.</code>' (dot) separators,</li>
+ * <li>Transition between characters and digits also constitutes a separator:
  *     <code>1.0alpha1 =&gt; [1, [alpha, 1]]</code></li>
- * <li>unlimited number of version components,</li>
- * <li>version components in the text can be digits or strings,</li>
- * <li>strings are checked for well-known qualifiers and the qualifier ordering is used for version ordering.
+ * <li>Unlimited number of version components,</li>
+ * <li>Version components in the text can be digits or strings,</li>
+ * <li>Strings are checked for well-known qualifiers, and the qualifier ordering is used for version ordering.
  *     Well-known qualifiers (case-insensitive) are:<ul>
  *     <li><code>alpha</code> or <code>a</code></li>
  *     <li><code>beta</code> or <code>b</code></li>
  *     <li><code>milestone</code> or <code>m</code></li>
  *     <li><code>rc</code> or <code>cr</code></li>
  *     <li><code>snapshot</code></li>
- *     <li><code>(the empty string)</code> or <code>ga</code> or <code>final</code></li>
+ *     <li><code>ga</code> or <code>final</code></li>
```

Comment:
Why removing the empty string ? That sounds like useful info wrt ordering.

Reply:
This indicates that when sorting, no qualifier will be sorted at the same level than `ga` or `final`.  I would not remove this bit.

Reply:
The empty string is not a qualifier. It's an absence of a qualifier. Not having a qualifier sorts the same as having a qualifier of "ga" or "final" buit I have trouble including it in a list of qualifiers.

Reply:
done